app/assets/stylesheets/user.css.scss

```diff
@@ -0,0 +1,25 @@
+.user-background {
+  padding-top: 1%;
+  padding-bottom: 1%;
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.